src/components/subscriptions/AddUsersButton.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import AddUsersModal from '../../containers/AddUsersModal';
+import ActionButtonWithModal from '../ActionButtonWithModal';
+
+const AddUsersButton = ({
+  onSuccess,
+  onClose,
+}) => (
+  <ActionButtonWithModal
+    buttonLabel="Add Users"
+    buttonClassName="add-btn btn btn-primary float-right"
+    renderModal={({ closeModal }) => (
+      <AddUsersModal
+        couponId="1"
+        title="Subscribe Users"
```

Comment:
Hmm 🤔 Perhaps "Add Users" so the title of the modal matches the action specified on the "Add Users" button.